create_hypothesis_user.py

```diff
@@ -0,0 +1,76 @@
+import requests
+from urllib.parse import urlparse
+import json
+
```

Comment:
This is the script we will effectively need to run for every ORCID that we add to a box.  This isn't the final resting place for this code as we will need some kind of user onboarding flow that this will need to tie into.  However, it lets us get started via a manual process while we spin up a demo environment.